index.js

```diff
@@ -68,7 +68,28 @@ function toggleColor(element) {
   }
 }
 
-/***** Deliverable 2 *****/
+document.querySelector("#header").addEventListener("click", function(e) {
+  toggleColor(e.target)
+})
 
+/***** Deliverable 2 *****/
+const like = document.querySelector('.like-button')
+like.addEventListener('click', function(){
+  const likes = document.querySelector('.likes')
+  const newlike = parseInt(likes.textContent) + 1
+  likes.textContent = newlike + ' likes'
+})
```

Comment:
The key here is that you have to read some information from the DOM to get the current number of likes, increment it, and then write it back to the DOM. Nice work!